### PR TITLE
task #23 Vnaida/homework lesson 23 lambda homework

### PR DESCRIPTION
**Pull request by VNaida33**

### CODE REVIEW

File: NUnitTests/Lessons/23/LambdaHomework.cs [hunk 1]
```diff
@@ -52,73 +52,87 @@ public void Test_FilterEvenNumbers_Function()
...
         public void Test_Where_NoLambda()
         {
             var myList = new List<string> { "one", "two", "three", "four" };
             var filterredList = new List<string>();
 
-            ...
-
+            foreach (var item in myList)
+            {
+                if (item.Length == 3)
```
Comment (vldemyan) [inline NUnitTests/Lessons/23/LambdaHomework.cs:104]: by "Uncomment and implement without lambda functions" I meant not just some random lambda which fits Assert, but to rewrite previous test Test_Where_Lambda without lambda here

File: NUnitTests/Lessons/23/LambdaHomework.cs [hunk 2]
```diff
@@ -52,73 +52,87 @@ public void Test_FilterEvenNumbers_Function()
...
         {
             var myList = new List<string> { "one", "two", "three", "four" };
-            bool result
-
-            ...
+            bool result = true;
 
+             foreach (var item in myList)
+            {
+                if (item.Length <= 2)
```
Comment (vldemyan) [inline NUnitTests/Lessons/23/LambdaHomework.cs:129]: the same here. I meant condition "Length > 0"

**Review by vldemyan — APPROVED** [review]

File: NUnitTests/Lessons/23/LambdaHomework.cs (hunk 2 above)
Comment (VNaida33) [reply NUnitTests/Lessons/23/LambdaHomework.cs:129]: @vldemyan i'll update the tests to be aligned with logic.

File: NUnitTests/Lessons/23/LambdaHomework.cs (hunk 1 above)
Comment (VNaida33) [reply NUnitTests/Lessons/23/LambdaHomework.cs]: fixed

File: NUnitTests/Lessons/23/LambdaHomework.cs (hunk 2 above)
Comment (VNaida33) [reply NUnitTests/Lessons/23/LambdaHomework.cs]: fixed

**Pull request merged**